gui/builtinContextMenus/shipModeChange.py

```diff
@@ -32,7 +37,12 @@ def getText(self, callingWindow, itmContext):
         return _t("Tactical Mode")
 
     def addMode(self, menu, mode):
-        label = mode.item.name.rsplit()[-2]
+        label = None
+        for raw, locaized in self.modeMap.items():
+            if raw in mode.item.typeName:
+                label = locaized
+                break
+
```

Comment:
I think this can be simplified using the original code and accessing the dictionary using it:

```
key = mode.item.typeName.rsplit()[-2]
label = self.modeMap[key]
```

This would avoid a double loop :)

Reply:
Did not think that way. Let me change it real quick.